start-project-template-inventory.json

```diff
@@ -0,0 +1,64 @@
+{
+    "bff-with-react": {
+        "coreTemplates": [ 
+            { "name": "im-build-dotnet-ci.yml", "canHaveMultiples": false},
+            { "name": "im-deploy-az-app-manually.yml", "canHaveMultiples": true},
+            { "name": "im-deploy-multiple-items-at-once.yml", "canHaveMultiples": false},
+            { "name": "im-deploy-cleanup-automated-board.yml", "canHaveMultiples": false},
+            { "name": "im-test-checkmarx-cli-sast-scan.yml", "canHaveMultiples": false}
+
+        ],
+        "optionalTemplates": [
```

Comment:
Do users get to select which of these they want during the yeoman generation process?

Reply:
They will get the core items by default and they are asked if they want the optional ones which they will get all or nothing from that group.  I'm probably going to do another PR in this repo that gives a better summary at the top of each workflow (that matches the git-er-done description) and also include what it's a core/optional template for.  That way when it gets pulled over in start-project, the user will have a better idea of what they can delete and what they should keep.